backend/quests/inputs.go

```diff
@@ -8,6 +8,11 @@ type PixelQuestInputs struct {
 	Color        uint8
 }
 
+type VoteQuestInputs struct {
+	DayIndex uint32
+	Color    uint8
```

Comment:
The `Inputs` type is for init params when calling the constructor of the vote quest.

So here, `VoteQuestInputs` will only take the `DayIndex`. ie `Color` can be removed